news/2018-12-29-the-qat-gazette-december-2018.md

```diff
@@ -0,0 +1,83 @@
+
+### New Beatmap Nominator Promotions
+
+With the potential for significant change coming to the QAT over the coming weeks (as will be discussed later in this article), there will naturally be an uncertainty for future Beatmap Nominator applications. Regular scheduling would have the next cycle start in mid- to late-January, but at present, the team is debating whether these can (or should) be operated alongside the QAT restructure. We will bring you a more clear update on this matter once it has been decided upon.
+
+### Beatmap Nominator Retirements
+
+With the prospect of new Nominator applications to hopefully be announced soon, we must also take a moment to acknowledge the lasting contributions of those who have since moved on. Please show your appreciation for [hikiko-](https://osu.ppy.sh/users/6512678), [Wishkey](https://osu.ppy.sh/users/1101704) and [Yauxo](https://osu.ppy.sh/users/898306) – may their future endeavours be as prosperous as their time in osu!.
+
+## Within the Quality Assurance Team
+
+As mentioned previously, unprecedented levels of change could be just over the horizon for the Quality Assurance Team, so the final days of 2018 bring both apprehension and excitement in equal measure. There's undeniably a lot to take in over the following few paragraphs, but if you want to witness (or even better, participate in!) the next stage of osu! mapping evolution, it'll be well worth the read!
+
+Before we get stuck in however, please give a show of appreciation for current Quality Assurance Team Leader [Mao](https://osu.ppy.sh/users/2204515), who joined an elite club recently in racking up an incredible *four years of service* in the team. We sincerely hope Mao continues to retain his boundless enthusiasm as he prepares to lead the QAT through the coming change.
+
+### Proposed changes to the QAT
+
+As mentioned several times in this article already, significant changes could be just on the horizon for the Quality Assurance Team, which may drastically change how we operate in the future. You can find more details on this announcement and the reasons for it in the Development forum, where [discussions towards multiple osu!team and community proposals](https://osu.ppy.sh/community/forums/topics/842601) are currently taking place.
+
+The first of these changes has been one of leadership, with [-Mo-](https://osu.ppy.sh/u/2202163) replacing myself as QAT Leader, alongside the continuing [Mao](https://osu.ppy.sh/users/2204515). Whilst I have thoroughly enjoyed my time at the head of the QAT, my free time available for osu! has been reduced considerably over the past few months, and I no longer feel I can properly fulfil the expectations of my role, or the community who elected me to it. On a positive note, new faces provide the potential for new direction, so please support -Mo- warmly as he settles into the challenge of management!
```

Comment:
-Mo- link should use /users/ instead of /u/

Reply:
Whoops, fixed